Makefile

```diff
@@ -46,11 +46,7 @@ include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
 # You can list all codegen related variables by:
 #   $ make -n --print-data-base | grep ^CODEGEN
 .PHONY: image-build
-# let's disable generate for for now
-# it updates libs and I think it is better to do that manually
-# especially when changes will be backported
-#image-build: generate
-image-build:
```

Comment:
can we leave this the way it was previously, and catch up with alexander when he's back about how to follow up on this detail?

Reply:
I missed the line that go get CDI. it should be removed as well for the same reasons. 

The reason I had to remove the go get KV, is that it start fail the CI after KV release v1.2.0. The same will happend after the next release of CDI.

As I see it, removing the `make generate` is not the right fix here. We need to remove all the go get from the automation.

Reply:
I removed the last commit to undo the restoring of the `make generate`. I'll try to fix the rest in a following PRs.